diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+activate w
+w --> w: check authorization request signature
+w --> w: check on trustworthiness of Verifier (approach on trust mechanism)
+w --> rp: POST **request_uri** (\n[OPTIONALstate,\n[OPTIONAL]issuer, \n[OPTIONAL]wallet_metadata, \n[OPTIONAL]w_nonce, \n[OPTIONAL]w_ephm_key)
```

Comment:
6. Remove `issuer` in this step since the `issuer` is REQUIRED in `wallet_metadata`.
7. Remove `w_ephm_key` from step 6 since `jwks`/`jwks_uri` can be included in `wallet_metadata`. The wallet acts as an AS and makes their keys discoverable.
8. Rename `w_nonce` to `as_nonce`.

Reply:
> 7\. Remove `w_ephm_key` from step 6 since `jwks`/`jwks_uri` can be included in `wallet_metadata`. The wallet acts as an AS and makes their keys discoverable.

Given your proposal (6) that would mean a wallet wanting to receive an encrypted presentation using an ephm key request MUST NOT use the `issuer` based metadata discovery. We might also consider to allow for both parameters `issuer` and `wallet_metadata` with the rule that `wallet_metadata` parameters preced or MUST be disjunct from data obtained from the well-known location. WDYT?

Reply:
> 8\. Rename `w_nonce` to `issuer_nonce` (as in AS provided), or `pmx_nonce` (as in referring to the post metadata exchange request URI mode).

done

Reply:
> > 7. Remove `w_ephm_key` from step 6 since `jwks`/`jwks_uri` can be included in `wallet_metadata`. The wallet acts as an AS and makes their keys discoverable.
> 
> Given your proposal (6) that would mean a wallet wanting to receive an encrypted presentation using an ephm key request MUST NOT use the `issuer` based metadata discovery. We might also consider to allow for both parameters `issuer` and `wallet_metadata` with the rule that `wallet_metadata` parameters preced or MUST be disjunct from data obtained from the well-known location. WDYT?

I provided a suggestion today, see below. It is easier to discuss below.